docs/roles.md

```diff
@@ -39,6 +40,10 @@ Only the owner of an operator can execute these functions:
 Only the owner of a cluster can execute these functions:
 
 - `SSVNetwork.registerValidator` - Registers a new validator on the SSV Network
+- `SSVNetwork.bulkRegisterValidator` - Registers a set of validators in the same cluster on the SSV Network
 - `SSVNetwork.removeValidator` - Removes an existing validator from the SSV Network
+- `SSVNetwork.bulkRemoveValidator` - Bulk removes an set of existing validators in the same cluster from the SSV Network
```

Comment:
Please update to "Bulk removes a set of existing validators in the same cluster from the SSV Network"